debian/patches/0004-remove-absolute-lib-include-paths.patch

```diff
@@ -0,0 +1,12 @@
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -47,6 +47,6 @@
+ endif()
+ 
+ install(TARGETS smu
+-	ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+-	LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+-	PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}")
++	ARCHIVE DESTINATION "lib"
++	LIBRARY DESTINATION "lib"
++	PUBLIC_HEADER DESTINATION "include")
```

Comment:
The problem is that ```${CMAKE_INSTALL_INCLUDEDIR}``` resolves to a full path. So if the Debian package is being built in ```/foo/bar``` it resolves to something like ```/foo/bar/debian/tmp/foo/bar/usr``` etc. Ideally it should resolve to ```debian/tmp/usr``` and so on.  This causes the Debian packaging scripts (dh_install) to fail as it looks for files under ```debian/tmp/```.  This can remain a Debian only patch in the ```debian/patches``` directory if so desired. I wonder if there are more compatible ways to implement the CMake scripts so it builds on all distros, only using ```${CMAKE_INSTALL_PREFIX}``` to customise the build paths. Unfortunately I may not have the time to investigate this further. There is an additional issue mentioned in the ```debian/TODO``` file, that is spaces in directory names break the ```install``` target. Again I am not sure when I would be able to look into this.